charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -73,34 +70,47 @@ spec:
           value: http://$(HOST_IP):8500
           {{- end }}
         command:
-          - "/bin/sh"
-          - "-ec"
-          - |
-            consul-api-gateway server \
-              -sds-server-host {{ template "consul.fullname" . }}-api-gateway-controller.{{ .Release.Namespace }}.svc \
-              -k8s-namespace {{ .Release.Namespace }} \
-              {{- if .Values.global.enableConsulNamespaces }}
-              {{- if .Values.apiGateway.consulNamespaces.consulDestinationNamespace }}
-              -consul-destination-namespace={{ .Values.apiGateway.consulNamespaces.consulDestinationNamespace }} \
-              {{- end }}
-              {{- if .Values.apiGateway.consulNamespaces.mirroringK8S }}
-              -mirroring-k8s=true \
-              {{- if .Values.apiGateway.consulNamespaces.mirroringK8SPrefix }}
-              -mirroring-k8s-prefix={{ .Values.apiGateway.consulNamespaces.mirroringK8SPrefix }} \
-              {{- end }}
-              {{- end }}
-              {{- end }}
-              -log-level {{ default .Values.global.logLevel .Values.apiGateway.logLevel }} \
+        - "/bin/sh"
+        - "-ec"
+        - |
+          consul-api-gateway server \
+            -sds-server-host {{ template "consul.fullname" . }}-api-gateway-controller.{{ .Release.Namespace }}.svc \
+            -k8s-namespace {{ .Release.Namespace }} \
+            {{- if .Values.global.enableConsulNamespaces }}
+            {{- if .Values.apiGateway.consulNamespaces.consulDestinationNamespace }}
+            -consul-destination-namespace={{ .Values.apiGateway.consulNamespaces.consulDestinationNamespace }} \
+            {{- end }}
+            {{- if .Values.apiGateway.consulNamespaces.mirroringK8S }}
+            -mirroring-k8s=true \
+            {{- if .Values.apiGateway.consulNamespaces.mirroringK8SPrefix }}
+            -mirroring-k8s-prefix={{ .Values.apiGateway.consulNamespaces.mirroringK8SPrefix }} \
+            {{- end }}
+            {{- end }}
+            {{- end }}
+            -log-level {{ default .Values.global.logLevel .Values.apiGateway.logLevel }} \
```

Comment:
I know you didnt change this bit, but out of curiosity I wonder if apigw supports -log-json?

Reply:
Yeah looks like they support `-log-json` could we add it?

Reply:
done!